src/data/print.css

```diff
@@ -0,0 +1,8 @@
+table.summary td {
+	padding-left: 13px;
```

Comment:
This is because of the content editable thing, so that the fields are aligned with the value.